sal_interfaces/Dome/Dome_Commands.xml

```diff
@@ -1,241 +1,133 @@
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_crawlAz</EFDB_Topic>
+        <Description>Move the azimuth axis at constant velocity.</Description>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_crawlEl</EFDB_Topic>
+        <Description>Move the elevation axis (light/wind screen) at constant velocity.</Description>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_moveAz</EFDB_Topic>
+        <Description>Move the azimuth axis at constant velocity, starting from a specified position.</Description>
+        <item>
+            <EFDB_Name>azimuth</EFDB_Name>
+            <Description>The position.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_moveEl</EFDB_Topic>
+        <Description>Move the elevation axis (light/wind screen) at constant velocity, starting from a specified position.</Description>
+        <item>
+            <EFDB_Name>elevation</EFDB_Name>
+            <Description>The position.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_park</EFDB_Topic>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_setLouvers</EFDB_Topic>
+        <Description>Move one or more louvers.</Description>
+        <item>
+            <EFDB_Name>louvers</EFDB_Name>
+            <Description>Bitmask of louvers to move.</Description>
+            <IDL_Type>int</IDL_Type>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>

```

Comment:
Interesting change from ID to bitmask. Should this refer to an enum like in the events file?

Reply:
Yes. Which is why I asked you for the names of the louvers. I wanted to make the bitmask.

Reply:
My assumption was to do the translation from numeric IDs to the louvers matrix IDs inside the Dome CSC. However, I have sent the EIE documentation regarding the matrix IDs to you.